qiime/metadata.py

```diff
@@ -14,12 +16,30 @@ def __init__(self, dataframe):
     def __repr__(self):
         return self._dataframe.to_json()
 
+    @staticmethod
+    def load(path):
+        try:
+            df = pd.read_csv(path, sep='\t', index_col=0, dtype=object)
+        except pd.io.common.CParserError:
```

Comment:
Does this error get raised if `path` doesn't exist or has the wrong permissions?